invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/SelectedMembers.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Segment, Label, Icon, Header } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_communities/i18next";
+import { Image } from "react-invenio-forms";
+import _isEmpty from "lodash/isEmpty";
+
+export class SelectedMembers extends Component {
+  removeMember = (id) => {
+    const { selectedMembers, updateSelectedMembers } = this.props;
+    const newSelectedMembers = selectedMembers.filter(
+      (element) => element.id != id
+    );
+    updateSelectedMembers(newSelectedMembers);
+  };
+
+  render() {
+    const { selectedMembers } = this.props;
+    return !_isEmpty(selectedMembers) ? (
+      <>
+        <Header as="h4" className="ml-20">
+          {i18next.t("Selected members and groups:")}
```

Comment:
if no selected members we should display info about nothing selected yet, instead of hiding the header